app/services/userService.js

```diff
@@ -1,57 +1,48 @@
-const UserModel = require('../models').user;
 const jwt = require('jsonwebtoken');
 const bcrypt = require('bcryptjs');
+const UserModel = require('../models').user;
 const logger = require('../logger');
 const filterEmail = require('../utils/filterEmail');
 
 /**
  *
- * @param {string} password
+ * @param {string} password password
  * @returns {Promise} hashed password
  */
```

Comment:
We do not use this kind of documentation for every project. It is not wrong, of course, but it is no needed either.

Reply:
If any kind of project requires this level of documentation / formalism, it will probably use TypeScript instead of pure Node.